docs/en/stack/ml/df-analytics/evaluatedf-api.asciidoc

```diff
@@ -99,3 +99,38 @@ closer to 1, the better the algorithm performance.
 The {evaluatedf-api} can return the false positive rate (`fpr`) and the true 
 positive rate (`tpr`) at the different threshold levels, so you can visualize 
 the algorithm performance by using these values.
+
+
+[discrete]
+[[ml-dfanalytics-regression-evaluation]]
+=== {regression-cap} evaluation
+
+This evaluation type is suitable for evaluating {regression} models. The 
+{regression} evaluation type offers the following metrics to evaluate the model 
+performance:
+
+* Mean squared error (MSE)
+* R-squared (R^2^)
+
+
+[discrete]
+[[ml-dfanalytics-mse]]
+==== Mean squared error
+
+You can measure how well the {reganalysis} model is performing by computing the 
```

Comment:
It's a little unclear to me here whether we're saying that the user has to go off and calculate this value, or whether the API does it for them.  I think it's the latter.  If so, we can be clearer by doing something like this:
```suggestion
The API provides a MSE by computing the 
```